install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1510,6 +1510,8 @@ hubble:
     # type: [null, string]
     # @schema
     # -- Dial timeout to connect to the local hubble instance to receive peer information (e.g. "30s").
+    #
+    # This option has been deprecated and is a no-op.
```

Comment:
one more thing: can you add a note in `Documentation/operations/upgrade.rst` please? Thanks!

Reply:
Yes, I can. I'll do it on Monday. Have a nice weekend.